### ISSUE TITLE
Trailing whitespace in messages

### ISSUE THREAD

**Issue by Nikerabbit** [EOL/eol_website#73]
Translatewiki.net does not support trailing whitespace in messages. Those should be removed if not needed or another workaround to be found if cannot be removed (but do note translations already do not have the space, so they are broken).

Here is a list:
```
EOL:Website-about.trait bank.body html/en
'<p>The technical and data management  | '<p>The technical and data management 
team at EOL refer to the structured    | team at EOL refer to the structured   
data resources in EOL- trait and other | data resources in EOL- trait and other
attribute data, ecological             | attribute data, ecological            
interactions, and our native           | interactions, and our native          
classification, the Dynamic Working    | classification, the Dynamic Working   
Hierarchy- collectively as TraitBank.  | Hierarchy- collectively as TraitBank. 
This content is managed somewhat       | This content is managed somewhat      
differently than our "classic"         | differently than our "classic"        
content- images, video, audio and      | content- images, video, audio and     
descriptive text- and it's useful for  | descriptive text- and it's useful for 
different things.</p><p>TraitBank is   | different things.</p><p>TraitBank is  
younger than the rest of EOL; it was   | younger than the rest of EOL; it was  
originally conceived by Species Pages  | originally conceived by Species Pages 
Group director Cynthia Parr, and went  | Group director Cynthia Parr, and went 
live in EOL v2 in January 2014. Since  | live in EOL v2 in January 2014. Since 
then, structured data has been our     | then, structured data has been our    
highest priority for content           | highest priority for content          
development.</p><p>Now, you can <a     | development.</p><p>Now, you can <a    
href='%{example_search_path}'>search</ | href='%{example_search_path}'>search</
a> or browse EOL by structured data,   | a> or browse EOL by structured data,  
or download it, or you may encounter   | or download it, or you may encounter  
it on our new overview tab in the form | it on our new overview tab in the form
of <a                                  | of <a                                 
href='%{autogen_page_path}'>autogenera | href='%{autogen_page_path}'>autogenera
ted text</a>. Like the rest of EOL     | ted text</a>. Like the rest of EOL    
content, it is available for re-use,   | content, it is available for re-use,  
and comes with detailed attribution    | and comes with detailed attribution   
data so you can credit it if you use   | data so you can credit it if you use  
it in your own work.</p><p>This data   | it in your own work.</p><p>This data  
is modeled as a <a                     | is modeled as a <a                    
href='%{property_graph_url}'>property  | href='%{property_graph_url}'>property 
graph</a> in <a                        | graph</a> in <a                       
href='%{neo4j_url}'>neo4j</a>. <a      | href='%{neo4j_url}'>neo4j</a>. <a     
href='%{data_services_path}'>Data      | href='%{data_services_path}'>Data     
services</a> are available.</p> '      | services</a> are available.</p>'      

EOL:Website-cms page date/en
' Last Modified at: '                  | ' Last Modified at:'                  

EOL:Website-support.array.words connector/en
', '                                   | ','                                   

EOL:Website-support.array.two words connector/en
' and '                                | ' and'                                

EOL:Website-support.array.last word connector/en
', and '                               | ', and'                               

EOL:Website-term.name.by uri.http://eol(dot)org/schema/terms/ciliary mucus feeder/en
'ciliary-mucus feeder '                | 'ciliary-mucus feeder'                

EOL:Website-term.name.by uri.http://eol(dot)org/schema/terms/skeletal density=20/en
'skeletal density '                    | 'skeletal density'                    
```

**Comment by mvitale** [comment]
Thanks for bringing this up, @Nikerabbit. I've removed the trailing whitespace from all of these except for the support/connector keys, for which the whitespace is significant and necessary. Is there a way to mark these as ignored by Translatewiki?

Also, I committed these changes to our `main` branch, which has replaced `master`. I realize now that we neglected to notify Translatewiki of this change. Is there something I need to configure? Thanks!

**Comment by Nikerabbit** [comment]
What do you mean by marking as ignored? Marking messages as ignored for us means they are not translatable.

I suppose github has some redirect in place from master to main? But yes I can update that in our config.

**Comment by mvitale** [comment]
Yes, that's what I mean. If we do that (make the messages untranslatable), will manual changes to these values in the locale files be overwritten by subsequent Translatewiki updates?

I'm pretty sure there isn't a redirect from master -> main. Thanks for updating!

**Comment by Nikerabbit** [comment]
Untranslatable means that our system will remove the existing translations (as they are now ignored by the system) from the translation files on every export.

**Comment by mvitale** [comment]
In that case, I'll move these to a separate set of locale files so that we can manage them externally. Thanks.

**Comment by mvitale** [comment]
Fixed in 0c205693. @Nikerabbit , feel free to reopen if I missed anything.
